pkg/fake/ec2api.go

```diff
@@ -150,7 +162,34 @@ func (e *EC2API) CreateFleet(_ context.Context, input *ec2.CreateFleetInput, _ .
 				if skipInstance {
 					continue
 				}
-				amiID := aws.String("")
+				amiID := lo.ToPtr("")
```

Comment:
This function is getting quite large -- is there a way to sub-class some of this behavior into its own functions?

Reply:
Nope don't feel strongly -- probably should have marked this as optional in retrospect